Ryujinx.HLE/HOS/Services/Fs/FileSystemProxy/IFileSystem.cs

```diff
@@ -298,25 +181,19 @@ public ResultCode GetFileTimeStampRaw(ServiceCtx context)
         {
             string name = ReadUtf8String(context);
 
-            try
-            {
-                FileTimeStampRaw timestamp = _fileSystem.GetFileTimeStampRaw(name);
+            Result rc = _fileSystem.GetFileTimeStampRaw(out FileTimeStampRaw timestamp, name);
```

Comment:
```suggestion
            Result resultCode = _fileSystem.GetFileTimeStampRaw(out FileTimeStampRaw timestamp, name);
```
and add an empty line after.